### PR TITLE
docs(legal): merge rate limits

### PR DESCRIPTION
**Pull request by mffap**
- Editorial changes
- Move metrics into the rate limit policy (only applicable for ZITADEL Cloud)
- Add information about raising limits

```[tasklist]

### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

**Comment by mffap** [comment]
@eliobischof what do you think? are you fine with reviewing the changes?

@fforootd fyi

File: docs/docs/legal/rate-limit-policy.md [hunk 1]
```diff
@@ -12,18 +12,31 @@ To ensure the availability of our Services and to avoid slow or failed requests
...
 
 If you are blocked you will receive a `http status 429`.
 
-:::tip
+:::tip Implement exponential backoff
 You should consider to implement [exponential backoff](https://en.wikipedia.org/wiki/Exponential_backoff) into your application to prevent a blocking loop.
 :::
 
+:::info Raising limits
+We understand that there are certain scenarios where you must access ZITADEL from one IP Address, for example access behind a corporate proxy or simply many users behind a shared internet connection.
```
Comment (eliobischof) [inline docs/docs/legal/rate-limit-policy.md:26]: > or simply many users behind a shared internet connection

You mean NAT?

File: docs/docs/legal/rate-limit-policy.md [hunk 2]
```diff
@@ -12,18 +12,31 @@ To ensure the availability of our Services and to avoid slow or failed requests
...
+Please [get in touch](https://zitadel.com/contact) with us for to discuss your requirements and we'll find a solution.
```
Comment (eliobischof) [inline docs/docs/legal/rate-limit-policy.md:27]:
```suggestion
Please [get in touch](https://zitadel.com/contact) with us to discuss your requirements and we'll find a solution.
```

File: docs/docs/legal/rate-limit-policy.md (hunk 1 above)
Comment (mffap) [reply docs/docs/legal/rate-limit-policy.md:26]: > > or simply many users behind a shared internet connection
> 
> You mean NAT?

Probably... can you make a suggestion? I think mine will not sound as good.

Comment (eliobischof) [reply docs/docs/legal/rate-limit-policy.md:26]:
```suggestion
We understand that there are certain scenarios where your users access ZITADEL from shared IP Addresses.
For example if you use a corporate proxy or Network Address Translation NAT.
```

**Review by eliobischof — CHANGES_REQUESTED** [review]

**Review by eliobischof — APPROVED** [review]

**Pull request merged**